aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -623,14 +692,20 @@ module AptosFramework::Stake {
         let validator_set_config = borrow_global<ValidatorSetConfiguration>(@CoreResources);
         validate_lockup_time(stake_pool.locked_until_secs, validator_set_config);
 
-        // Throw an error is the validator is already active.
+        // Throw an error is the validator is already active, pending_active, or pending_inactive.
```

Comment:
the function name `is_validator_active` is confusing, probably should be `is_validator_inactive` and negates the boolean

Reply:
Why is it confusing?

Reply:
This will be changed in https://github.com/aptos-labs/aptos-core/pull/1860